packages/calcite-components/src/components/table-row/resources.ts

```diff
@@ -0,0 +1,3 @@
+export const CSS = {
+  isInTableHead: "is-in-table-head",
```

Comment:
hmm need a better name for this class. Any ideas?

Reply:
This one wasn't being used - removed.